local-libs/traceviewer-libs/react-components/style/react-contextify.css

```
@@ -2,233 +2,240 @@
     background-color: var(--trace-viewer-menu-selectionBackground);
-  }
-  .react-contexify__theme--light .react-contexify__item__content {
+}
+.react-contexify__theme--light .react-contexify__item__content {
     color: var(--trace-viewer-menu-foreground);
-  }
-  
-  @keyframes react-contexify__scaleIn {
+}
+
+@keyframes react-contexify__scaleIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "react-contexify__scaleIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)